.github/workflows/gh-ci.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
       run: |
         PYTEST_FLAGS="--disable-pytest-warnings --durations=50"
         if [ ${{ matrix.codecov }} = "true" ]; then
-          PYTEST_FLAGS="${PYTEST_FLAGS} --cov=MDAnalysis --cov-report=xml"
+          PYTEST_FLAGS="${PYTEST_FLAGS} --cov-config=.coveragerc --cov=MDAnalysis --cov-report=xml"
```

Comment:
So it's not strictly necessary but we have multiple coveragerc files knocking about in the repo (I have another PR open looking to fix that) and I want to make 100% sure we're picking up the right thing.

Reply:
Better explicit than implicit.

(I was thinking even `./.coveragerc` but that seems too much.)